docs/app/pages/landing/landing.component.html

```diff
@@ -2,6 +2,13 @@
 
 <uxd-full-page-layout>
 
+    <ux-alert type="warning">
```

Comment:
This is grand, however I would suggest using the error style for alert because orange on white is quite difficult to read. I think that also corresponds to the increased urgency of this announcement.